From 191f89cd412a4e89687728e5af80ef8cf3024d5c Mon Sep 17 00:00:00 2001
From: Aisheng.Dong <b29396@freescale.com>
Date: Tue, 25 Jan 2011 19:29:48 +0800
Subject: [PATCH] ENGR00138491 mx51/mx50: enable SD2 power during suspend

Remove the regulator standby mask for sd2 to make sure its power
is enabled during suspend.
This is required by WiFi deep sleep support.

Signed-off-by: Aisheng.Dong <b29396@freescale.com>
---
 arch/arm/mach-mx5/mx50_rdp_pmic_mc13892.c     |    4 ++--
 arch/arm/mach-mx5/mx51_babbage_pmic_mc13892.c |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-mx5/mx50_rdp_pmic_mc13892.c b/arch/arm/mach-mx5/mx50_rdp_pmic_mc13892.c
index 883108a..b362a5a 100644
--- a/arch/arm/mach-mx5/mx50_rdp_pmic_mc13892.c
+++ b/arch/arm/mach-mx5/mx50_rdp_pmic_mc13892.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2010 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright (C) 2010-2011 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -77,7 +77,7 @@
 #define REG_MODE_0_ALL_MASK	(DIG_STBY_MASK | GEN1_STBY_MASK\
 					| PLL_STBY_MASK | IOHI_STBY_MASK)
 #define REG_MODE_1_ALL_MASK	(CAM_STBY_MASK | VIDEO_STBY_MASK |\
-				AUDIO_STBY_MASK | SD_STBY_MASK | GEN3_STBY_MASK)
+				AUDIO_STBY_MASK | GEN3_STBY_MASK)
 
 /* switch mode setting */
 #define	SW1MODE_LSB	0
diff --git a/arch/arm/mach-mx5/mx51_babbage_pmic_mc13892.c b/arch/arm/mach-mx5/mx51_babbage_pmic_mc13892.c
index 1d5a06b..89a7514 100644
--- a/arch/arm/mach-mx5/mx51_babbage_pmic_mc13892.c
+++ b/arch/arm/mach-mx5/mx51_babbage_pmic_mc13892.c
@@ -2,7 +2,7 @@
  * mx51-babbage-pmic-mc13892.c  --  i.MX51 Babbage Driver for Atlas MC13892 PMIC
  */
  /*
-  * Copyright (C) 2009-2010 Freescale Semiconductor, Inc. All Rights Reserved.
+  * Copyright (C) 2009-2011 Freescale Semiconductor, Inc. All Rights Reserved.
   */
 
  /*
@@ -72,7 +72,7 @@
 
 /* 0x92412 */
 #define REG_MODE_0_ALL_MASK	(GEN1_STBY_MASK |\
-				DIG_STBY_MASK | GEN2_STBY_MASK |\
+				DIG_STBY_MASK |\
 				PLL_STBY_MASK)
 /* 0x92082 */
 #define REG_MODE_1_ALL_MASK	(CAM_STBY_MASK | VIDEO_STBY_MASK |\
-- 
1.5.4.4

